src/ember-cli.ts

```diff
@@ -75,6 +75,8 @@ export class EmberCliManager {
                 return;
             };
 
+            result = fileNameSanitation(result);
```

Comment:
I was going to, but then realized the normal ember cli can create projects with caps and spaces, so I tried to wrap it in quotes to fix the problem. This is kind of OS + cmd vs powershell specific so I didn't like this approach and phased it out using a different method in my new branch using cross spawn which does all that fancy footwork.